wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -58,6 +67,26 @@ function TransferLayout(props: PromptRequestTransferData) {
   );
 }
 
+function SignLayout(props: PromptRequestCallSCData) {
+  const { GasLimit, Coins, Address, CalledFunction, OperationType } = props;
+
+  return (
+    <div>
+      <div>Operation Type: {OperationType}</div>
```

Comment:
`div`? why not `p`?

Reply:
the front end will be upgraded by the front experts , my code is just aa proof that it's working well